docker/test/integration/runner/Dockerfile

```diff
@@ -6,8 +6,8 @@ ARG apt_archive="http://archive.ubuntu.com"
 
 RUN sed -i "s|http://archive.ubuntu.com|$apt_archive|g" /etc/apt/sources.list
 
-RUN apt-get update \
```

Comment:
Someone said previously that the `apt` command does not have a stable API, in contrast to `apt-get`.

Reply:
It was an attempt to download packages in parallel. https://twitter.com/ClickHouseSRE/status/1648601294372732930

I'll rather roll it back to not keep accidentally